ptx/sec_deriv_interpret.ptx

```diff
@@ -569,12 +569,13 @@
           <caption>Zooming in on <m>f</m> and its tangent line at <m>x=3</m> for the function given in <xref ref="ex_der_meaning4">Examples</xref> and <xref ref="ex_der_meaning5"/></caption>
           <!-- START figures/fig_fwithderivzoom3.tex -->
           <image xml:id="img_fwithderivzoom3" width="47%">
-          <shortdescription> Zooming in on f and its tangent line at x=3 for the function given in example 2.2.7 and 2.2.10 </shortdescription>
+          <shortdescription>Zooming in on f and its tangent line at x=3 for the function given in <xref ref="ex_der_meaning4">Examples</xref> and <xref ref="ex_der_meaning5"/></shortdescription>
```

Comment:
You can't use `xref` tags in the short description. You need to find another way to reference the examples that doesn't involve their numbers.

The only tag allowed in the short description is `var`, if you have a webwork image with variable parameters.